roles/network_plugin/calico/templates/calico.env.j2

```diff
@@ -4,6 +4,11 @@ ETCD_CERT_FILE="{{ calico_cert_dir }}/cert.crt"
 ETCD_KEY_FILE="{{ calico_cert_dir }}/key.pem"
 CALICO_IP="{{ip | default(ansible_default_ipv4.address) }}"
 CALICO_IP6=""
+{% if calico_network is defined and calico_network_backend == 'none' %}
```

Comment:
I don't see calico_network defined anywhere. Some doc on its usage would be good.